Configurations.md

```diff
@@ -2353,9 +2353,59 @@ Require a specific version of rustfmt. If you want to make sure that the
 specific version of rustfmt is used in your CI, use this option.
 
 - **Default value**: `CARGO_PKG_VERSION`
-- **Possible values**: any published version (e.g. `"0.3.8"`)
+- **Possible values**: `semver` compliant values, such as defined on [semver.org](https://semver.org/).
 - **Stable**: No (tracking issue: [#3386](https://github.com/rust-lang/rustfmt/issues/3386))
 
+#### `CARGO_PKG_VERSION` (default):
+
+Match on exact version
```

Comment:
Not sure if we need to specify ``#### `CARGO_PKG_VERSION` (default):`` Probably nicer to stay consistent with what you've done for the other examples, which I think are very clear and helpful!

```suggestion
#### Match on exact version
```